lib/query/woqlQuery.js

```diff
@@ -18,6 +18,57 @@ WOQLQuery.prototype.read_object = function(IRI, OutputVar, Format) {
     return this.wform(Format)
 }
 
+WOQLQuery.prototype.read_document = function(IRI, OutputVar, Format) {
```

Comment:
The definition of `WOQLQuery.prototype.read_object` no longer works, so, in the goal of keeping things working and compatible, can you delete the definition and add this?

```js
WOQLQuery.prototype.read_object = WOQLQuery.prototype.read_document
```

For clarity, that should probably go after the `WOQLQuery.prototype.read_document` definition.

Reply:
Rather than adding this here i think it would be better if i add this 
```
return new WOQLQuery().read_document(IRI, output, formatObj)
```
in here
https://github.com/terminusdb/terminusdb-client-js/blob/6d56272b37bfb5937017a88e491415ded25cebbf/lib/woql.js#L104

Reply:
Okay, but isn't `WOQLQuery.prototype.read_object` still wrong? We don't have `ReadObject` anymore.

Reply:
Yup so should i just remove it 😅

Reply:
Yep, just remove it. I confirmed with @Francesca-Bit that it's no longer accessible.